scripts/foundry/utils/Constants.s.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.17;
+
+import "contracts/utils/Constants.sol";
+
+/*//////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+                                                   MAINNET CONSTANTS                                                
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////*/
+
+address constant ACCESS_CONTROL_MANAGER = 0x5bc6BEf80DA563EBf6Df6D6913513fa9A7ec89BE;
```

Comment:
We can get inspired from what we have in the angle-multisig or angle-gpvernance repo to get them directly from our sdk even in Foundry. It saves a ton of time and less error prone

Reply:
Actually don't need it in the Foundry script so will just keep the addresses I need (create2 factory address and deployer), and next time we need this, will do with the SDK